Closes #9 Update readme, provide images, pcb and 3d qr adapter model

sabifa opened the pull request:
After quite a long time I finally can report something good 😄 
I managed to get it working reliably using an Arduino Pro Mini V2 which runs 3.3V at 16 MHz. 8 MHz does not seem to be fast enough.

I created a small PCB which holds the Pro Mini and a 3D printable QR adapter so one can make himself a emulator with a low profile. 

I've updated the readme in the PR (feel free to remove the top part if you don't like it) and a more in depth readme on how to get started with the Pro Mini V2. Let me know if I should change something or feel free to change it yourself 🙂

sabifa committed:
Closes #9 Update readme, provide images, pcb and 3d qr adapter model

lshachar commented:
Good job! I intend to have a look at it and include it in the source files. :) 👍

lshachar commented:
A few things,
in readme.md you wrote: *"In some rare cases my base reconnects if I connect the emulator to it"*, do you mean 
*'In some rare cases my base reconnects if I connect the pro mini to my pc over USB?'*

I also wonder about the 5v connection from the wheelbase to Raw through the diode. [I read that ](https://forum.arduino.cc/index.php?topic=144441.0) raw feeds the onboard regulator. do you actually need that connection? you already have vcc connected directly to 3.3v. did you try / can you try to remove that connection and see if it still works?

Also, there's an issue with having the FTDI connected while the pro mini is powered from wheelbase. since this will connect the 3.3v coming from the FTDI, with the 3.3v supply coming from the wheelbase. (same problem I had with the 5v line).
I see that in your How-To you specified to first program the pro mini, and later to connect it to the wheelbase. But this way the next guy couldn't work out any issues through the ftdi's serial interface once everything is put together.
One solution would be to write in **bold** that it is very important to remove the VCC pin on the FTDI interface (**at least** after the initial programing!) and to explain specifically about this hazard.

The issue you had with the funky switch always being pressed: since you're not using the analog Dpad, make sure to comment out the line `#define HAS_ANALOG_DPAD` in `arduino_fanatec_wheel.ino` or otherwise, the base thinks that the funky push button is always pushed. add that to your How to directions!

sabifa commented:
> in readme.md you wrote: "In some rare cases my base reconnects if I connect the emulator to it", do you mean
'In some rare cases my base reconnects if I connect the pro mini to my pc over USB?'

This happens from time to time if I connect the emulator (so the PCB) to my base **without** USB connected. Not too sure why this could happen, it felt pretty random. I even imagined that the speed of sliding the QR onto the wheelbase influences whether the base restarts or not 😅 

> I also wonder about the 5v connection from the wheelbase to Raw through the diode. I read that raw feeds the onboard regulator. do you actually need that connection? you already have vcc connected directly to 3.3v. did you try / can you try to remove that connection and see if it still works?

Good catch! I thought that connecting 5V to RAW might improve the voltage stability if the 3.3V from the wheelbase ever drops a bit. I will desolder the the diode to disconnect 5V to RAW and see if it still works.

> One solution would be to write in bold that it is very important to remove the VCC pin on the FTDI interface (at least after the initial programing!) and to explain specifically about this hazard.

That sounds like a good solution for now. I will update the readme!

Regarding the funky switch, I will also try that out and report back in the next days

sabifa committed:
#9 Update readme and PCB files

- add wheelbase damage warning
- add note to uncomment lines
- remove connection of 5V to RAW (including diode)

sabifa commented:
@lshachar You were totally right with your assumption that the connection from 5V to RAW is not necessary.
I've updated the pcb files & Gerber and also updated the readme based on your reply above.

Please let me know if I should change something

lshachar commented:
Hi @sabifa,
The 'Pro Mini V2' is actually, well, a lie. [Apparently the ATmega328 on it needs at least 3.78V to run properly at 16 MHz.](https://forum.arduino.cc/index.php?topic=300426.0) it isn't guranteed to work properly at 3.3V.
However, since it is working for you, I do intend to eventualy include it in the repository (with all the warnings that should go with it!)
Sorry for taking ages, I got too many things on my plate!
